client/source/components/questionsAnswers/SearchQuestions.jsx

```diff
@@ -1,16 +1,58 @@
-import React from 'react';
-import { InputContainer } from './styles/Containers.styled';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const SearchQuestions = function CreateSearchQuestions({ handleSearch }) {
+  const [input, setInput] = useState('');
+
+  // console.log('questions: ', questions);
 
-const SearchQuestions = function CreateSearchQuestionsComponent() {
   return (
     <InputContainer>
-      <input
+      <Input
         type="text"
         placeholder="Have a Question? Search For Answers"
+        onChange={(e) => setInput(e.target.value)}
+        value={input}
       />
-      <button type="button" onClick={() => {}}>Go!</button>
+      <Button
+        type="button"
+        onClick={() => {
```

Comment:
I wasn't able to figure out based on your initial comments if this is still WIP in terms of functionality. When I search a keyword that I see amongst the answers, and press Go, it doesn't return me the expected results of filtered results with input text.

Reply:
Hmm, now this one is weird. This looks like an old version of the SearchQuestions component, so maybe something didn't fetch, or VS code is acting weird. But, I can assure you that it has working functionality on my end. After I finish these updates, I will push again and we'll see if that is still happening on your end.

Reply:
Thanks for taking a look!

Some new observations: I searched the word "quod", and it filtered to the one question upon clicking Go. However, after removing the search word of "quod" from the search bar, it doesn't appear to restore the list to the previous state per the BRD requirement here: 

_If the user clears the search term, or removes characters so that less than 3 remain, the list should return to the state where it is not filtered to matching text._

Reply:
It's working, and looks good!